autotyping/guess_type.py

```diff
@@ -35,7 +35,7 @@ def guess_type_from_argname(name: str) -> Tuple[Optional[str], List[str]]:
     # e.g. set_of_widths => Set[int]
     m = re.fullmatch(
         rf"(?P<elems>\w*?)_?(?P<container>{containers})", name
```

Comment:
```suggestion
        rf"(?P<elems>\w+?)_?(?P<container>{containers})", name
```
This one too (I think this is actually the more important one, the other one would trigger only if your varname ends with `_of_`).

Reply:
thanks - do you have an example test case which would make it fail without this?

Reply:
hm, I would have expected the existing test case to fail with the version you had, so maybe I'm missing something

Reply:
ah got it, I was just missing `guess_common_names=True`, sorry